client/api/volume/v1beta3/api.proto

```diff
@@ -5,128 +5,129 @@ package v1beta3;
 option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1beta3";
 
 service Volume {
-    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for
-    // all volumes on a Disk device
+    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for all volumes on a Disk device.
     rpc ListVolumesOnDisk(ListVolumesOnDiskRequest) returns (ListVolumesOnDiskResponse) {}
-    // MountVolume mounts the volume at the requested global staging path
+    // MountVolume mounts the volume at the requested global staging path.
     rpc MountVolume(MountVolumeRequest) returns (MountVolumeResponse) {}
-    // DismountVolume gracefully dismounts a volume
-    rpc DismountVolume(DismountVolumeRequest) returns (DismountVolumeResponse) {}
-    // IsVolumeFormatted checks if a volume is formatted with NTFS
+    // UnmountVolume gracefully unmounts a volume.
```

Comment:
UnmountVolume flushes data cache to disk and removes the global staging path?

Reply:
fixed